lib/output.js

```diff
@@ -68,8 +69,10 @@ function OutputStream(options) {
         preamble         : null,
         quote_style      : 0,
         keep_quoted_props: false,
-        ecma             : 5,
+        ecma             : 6,
```

Comment:
Could you please revert back to `ecma: 5` as default? That way ES5-only code will still uglify correctly with harmony.